src/js/components/award/RecipientAddress.jsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * RecipientAddress.jsx
+ * Created by Emily Gullo 01/31/2017
+ **/
+
+import React from 'react';
+
+const propTypes = {
+    recipient: React.PropTypes.object,
+    type: React.PropTypes.string,
+    titleValue: React.PropTypes.string
+};
+
+export default class RecipientAddress extends React.Component {
+
+    render() {
+        const recipient = this.props.recipient;
+        const city = recipient.recipient_city;
+        const stateProvince = recipient.recipient_state_province;
+        let cityState = "";
+
+        if (city && stateProvince) {
+            cityState = `${city}, ${stateProvince}`;
+        }
+        else if (!city && !stateProvince) {
+            cityState = null;
+        }
+        else {
+            cityState = `${city} ${stateProvince}`;
```

Comment:
This seems to be handling if there is either city or state, but won't this return blank spaces (especially leading spaces if there's no city)?